docs/content/learn/tutorials/component-definition/component.md

```diff
@@ -0,0 +1,682 @@
+---
```

Comment:
This file should be named `_index.md`. Issue the command `git mv component.md _index.md` to rename it.